warehouse/packaging/tasks.py

```diff
@@ -508,3 +509,9 @@ def populate_data_using_schema(file):
                 json_rows, table_name, job_config=LoadJobConfig(schema=table_schema)
             ).result()
             break
+
+
+@tasks.task(ignore_result=True, acks_late=True)
+def generate_projects_simple_detail(request, projects):
```

Comment:
About this task

1. I need it to have the `request` object to be used by the `render_simple_detail` function. 
I couldn't find a way to retrieve the `request` , only using tasks.
You can see it is used during the process of the `warehouse tuf dev import-all` command https://github.com/kairoaraujo/warehouse/blob/706478ac2f375e6f2d03259ee75cba67c91f21ae/warehouse/cli/tuf.py#L136

2. I also intended to use it later during the **Manage packages/project and simple details to TUF** commit. Still, it wasn't possible because I could not send the simple detail generated in the background. I need the response to use in the TUF metadata.

If I can retrieve the `request` object or generate one for use with the `render_simple_detail`, we can remove it.